pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -31,6 +31,7 @@ func (f *frontend) putOrPatchOpenShiftCluster(w http.ResponseWriter, r *http.Req
 
 	var header http.Header
 	var b []byte
+	var identityURL []string
```

Comment:
nit: considering this is an array, might be better named `identityURLs`

Reply:
I'm going to go with this suggestion: https://github.com/Azure/ARO-RP/pull/3514#discussion_r1603751020 which should resolve this comment.